Fixing

zuckung opened the pull request.

Hecter94 commented:
Could you add some details about what it is you're fixing, why it needs to be fixed and what you're doing to fix it?

zuckung commented:
Thought the commit names express enough.

Removed the empty lines between news entries
Reduced news  entries to 10
Added a --- behind the news
Added category anchor link to new generated news

So, cosmetic changes mostly.

zuckung commented:
Moved readme generating script to ist own scheduled workflow (daily 4:30 am utc, 1 hour after the update check). 

Sometimes the readme script failed because somehow the http header of the asset zip cant be read(maybe something github specific?), giving github 1 hours to correctly update the zips many fix this.

Additional the on push workflow takes less time, so that you can do 2 plugin PRs faster. We hat some failed ones because 2 workflows cant run at the same time and doing the same.
